app/Http/Controllers/WithdrawalController.php

```diff
@@ -27,24 +27,17 @@
 
 class WithdrawalController extends Controller
 {
-    /**
-     * @return View
-     */
-    public function index()
+    public function index(): View
     {
         return view('omnomcom.withdrawals.index', ['withdrawals' => Withdrawal::orderBy('id', 'desc')->paginate(6)]);
     }
 
-    /** @return View */
-    public function create()
+    public function create(): View
     {
         return view('omnomcom.withdrawals.create');
     }
 
-    /**
-     * @return RedirectResponse
-     */
-    public function store(Request $request)
+    public function store(Request $request): RedirectResponse
```

Comment:
Why are all the comments removed?

Reply:
Pint does this when you actually use the typed function return